### PR TITLE
Support to build jni under MasOS

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: src/jni/CMakeLists.txt [hunk 1]
```diff
@@ -31,12 +35,33 @@ include_directories($ENV{JAVA_HOME}/include
...
-link_directories(
-    ${NEBULA_THIRDPARTY_ROOT}/lib
-    ${NEBULA_THIRDPARTY_ROOT}/lib64
-)
+
+if (NOT DEFINED ENV{JAVA_HOME})
+    message(FATAL_ERROR "Couldn't find JAVA_HOME")
+endif()
+
+if (APPLE)
```
Comment (dangleptr) [inline src/jni/CMakeLists.txt:44]: MacOS is better.

**Review by dangleptr — COMMENTED** [review]
BTW Is there anybody need this feature?

File: src/jni/CMakeLists.txt (hunk 1 above)
Comment (laura-ding) [reply src/jni/CMakeLists.txt:44]: `APPLE` is defined by cmake

**Review by critical27 — APPROVED** [review]
LGTM

**Review by darionyaphet — APPROVED** [review]
LGTM

**Pull request merged**